frontend/libs/studio-components/src/components/StudioModal/StudioModal.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Denne filen inneholder `act(() =>` i unit-testene. Stemmer for å rydde opp i dette så vi ikke drar med oss sånn "gjeld" inn i en fresh-pakke. 😊

Reply:
Får bare denne om jeg fjerner det 😞 
```
Warning: An update to ModalPortal inside a test was not wrapped in act(...).
    
When testing, code that causes React state updates should be wrapped into act(...):

act(() => {
  /* fire events that update state */
});
/* assert on the output */
```

Reply:
Ok, da må vi se på dette i en egen issue senere. :)